remote/metrics.go

```diff
@@ -122,6 +123,16 @@ func (i *instrumentedPlatform) SyncStatus(cursor string) (_ []string, err error)
 	return i.p.SyncStatus(cursor)
 }
 
+func (i *instrumentedPlatform) PublicSSHKey(regenerate bool) (_ ssh.PublicKey, err error) {
```

Comment:
This is mostly measuring (when regenerating keys) the time it takes for the remote fluxd to create a new key.
Is this what we want?
(I suppose there's no harm in collecting, I just think that the duration won't be very useful).

The count might be more interesting though.